### PR TITLE
Layout changes to the provider ui single application view

### PR DESCRIPTION
**Pull request by steventux**
## Context

Design changes in the [prototype app](https://manage-applications-beta.herokuapp.com/application/AS1623) should be applied to the application details page.

## Changes proposed in this pull request

- Move application status tag up in line with candidate name.
- Remove status box border.
- Move download as PDF link under candidate name.
- Move 'Respond to application' into a banner above candidate name.

#### After

image

## Link to Trello card

https://trello.com/c/zFHoIVCU/1783-layout-changes-to-the-provider-ui-single-application-view

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

**Comment by adamsilver** [comment]
Please move the download link under the h1:
image

**Comment by adamsilver** [comment]
Please move the first line to the summary list below it
Please delete the second line (as its covered by the banner)
image

**Comment by adamsilver** [comment]
Please make the offer styles and content match this:
image

**Comment by adamsilver** [comment]
We don't need a summary card around this anymore at all.

image

File: app/components/provider_interface/status_box_component.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div data-qa="app-status-box">
-  <div class="app-card app-card--white govuk-!-margin-bottom-9">
+  <div class="app-card app-card--white">
```
Comment (adamsilver) [inline app/components/provider_interface/status_box_component.html.erb]: We don't need a white summary card. Take the offer out of the box altogether.

File: app/frontend/styles/_card.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .app-card {
   background-color: govuk-colour("light-grey");
   display: block;
-  padding: govuk-spacing(2);
+  padding: govuk-spacing(2) 0 govuk-spacing(2) 0;
```
Comment (adamsilver) [inline app/frontend/styles/_card.scss]: Not sure any of these CSS changes are needed.

File: app/frontend/styles/_tag.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
 .app-tag {
```
Comment (adamsilver) [inline app/frontend/styles/_tag.scss]:
```suggestion
.govuk-heading-xl .app-tag {
```

File: app/views/provider_interface/application_choices/show.html.erb [hunk 4]
```diff
@@ -13,19 +13,37 @@
...
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-1">
-      <span class="govuk-caption-xl"><%= @application_choice.application_form.support_reference %></span>
       <%= @application_choice.application_form.full_name %>
+      <%= render(ProviderInterface::ApplicationStatusTagComponent.new(application_choice: @application_choice)) %>
     </h1>
+  </div>
+
+  <div class="govuk-grid-column-one-third">
     <p class="govuk-body govuk-!-margin-bottom-9 app-!-print-display-none">
       <%= govuk_link_to(
```
Comment (adamsilver) [inline app/views/provider_interface/application_choices/show.html.erb]: Move this under the h1

File: app/views/provider_interface/application_choices/show.html.erb [hunk 5]
```diff
@@ -13,19 +13,37 @@
...
+<div class="app-banner">
+  <div class="app-banner__message">
+    <h2 class="govuk-heading-m govuk-!-margin-bottom-0">
+      You have <%= days_to_respond_to(@application_choice) %> days to send a response. <%= govuk_link_to 'Respond to application', provider_interface_application_choice_respond_path(@application_choice) %>.
+    </h2>
+  </div>
+</div>
+<% end -%>
+
 <div class="govuk-grid-row">
```
Comment (adamsilver) [inline app/views/provider_interface/application_choices/show.html.erb]: Kill the grid stuff as the download link will go under the h1.

**Comment by steventux** [comment]
@stevehook Thanks for spotting this, I've since amended and tested with the timeline on and I think the position is correct now in that the timeline has the entire run of the third column to expand.

@adamsilver I've made the various amends requested, this should be good to re-review

**Comment by adamsilver** [comment]
image

Not sure why this is happening, you have the right margin bottom on the offer panel, but it's quite broken.

**Comment by steventux** [comment]
@adamsilver regarding https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1766#issuecomment-606593891 this is an unintentional side effect of conditionally showing the status box. Which also affects the positioning of the timeline.

**Pull request merged**